spec/integrations.yml

```diff
@@ -1,14 +1,14 @@
 ---
 - name: rubyspec
   namespace: Rubyspec
-  repo_uri: 'https://github.com/rubyspec/rubyspec.git'
+  repo_uri: 'https://github.com/ruby/rubyspec.git'
```

Comment:
Has development on the `rubyspec/rubyspec` repo stopped while the `ruby/rubyspec` one continued?